.github/workflows/docs.yml

```diff
@@ -18,13 +18,64 @@ name: "Documentation check"
 on:
 - pull_request
 
+env:
+  DEPS_BRANCH: bot/stable-deps-update
+
 jobs:
   sphinx:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v3
```

Comment:
# Question

Any particular reason we are not using [v4](https://github.com/actions/checkout)?
```suggestion
    - uses: actions/checkout@v4
```

Reply:
Sure, sounds reasonable to me!